docs/wallets.md

```diff
@@ -22,6 +22,19 @@ Supporting seed generation is a little more involved so the tradeoff between
 implementation complexity and user value is less clear, especially since the
 Codex provides users instructions on doing generation themselves.
 
+## Error Detection and Correction
+
+Wallets MUST support detection of errors using the codex32 checksum algorithm.
+Wallets MAY additionally support error correction; such wallets are referred to as "error-correcting wallets (ECWs)" and have additional requirements.
+
+An ECW
+
+* MUST support correction of up to 4 substitution errors;
+* MAY also support correction of up to 8 erasures;
```

Comment:
I don't see the problem here. Should I change line 32 to say "substitution or erasures" so that it's clear that any wallet that can correct 4 substitutions can also correct 4 erasures?

Reply:
That'd be more clear. But the detail I missed was that during correction without erasure correction support, '?' is treated as an arbitrary character. That's a bit different than these quotes:

BIP93
> Implementations interpret "?" as an erasure.

This doc
> ECWs MUST also allow entry of `?` which indicates an erasure (unknown character).

I'd say something along the lines of

"'?' can/may be substituted with an arbitrary bech32 character to correct up to 4 erasures as errors."